examples/nodejs/cache/doc-example-files/doc-examples-js-apis.ts

```diff
@@ -1481,64 +1481,64 @@ async function example_API_TopicSubscribe(topicClient: TopicClient, cacheName: s
   }
 }
 
-async function example_API_ListWebhooks(topicClient: TopicClient, cacheName: string) {
-  const result = await topicClient.listWebhooks(cacheName);
-  if (result instanceof ListWebhooks.Success) {
-    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-    console.log(`listed webhooks: ${result.getWebhooks()}`);
-  } else if (result instanceof ListWebhooks.Error) {
-    throw new Error(
-      `An error occurred while attempting to list webhooks for cache '${cacheName}': ${result.errorCode()}: ${result.toString()}`
-    );
-  }
-}
-
-async function example_API_DeleteWebhook(topicClient: TopicClient, cacheName: string) {
-  const result = await topicClient.deleteWebhook(cacheName, 'a webhook');
-  if (result instanceof DeleteWebhook.Success) {
-    console.log('successfully deleted webhook');
-  } else if (result instanceof ListWebhooks.Error) {
-    throw new Error(
-      `An error occurred while attempting to delete webhook 'a webhook' inside of cache '${cacheName}': ${result.errorCode()}: ${result.toString()}`
-    );
-  }
-}
-
-async function example_API_PutWebhook(topicClient: TopicClient, cacheName: string) {
-  const result = await topicClient.putWebhook(cacheName, 'examples webhook', {
-    topicName: 'a topic',
-    destination: 'https://www.thisisawebhookurl.com/v1/webhook',
-  });
-  if (result instanceof PutWebhook.Success) {
-    console.log('successfully created webhook');
-  } else if (result instanceof PutWebhook.Error) {
-    throw new Error(
-      `An error occurred while attempting to create a webhook 'examples webhook' inside of cache '${cacheName}': ${result.errorCode()}: ${result.toString()}`
-    );
-  }
-}
-
-async function example_API_RotateWebhookSecret(topicClient: TopicClient, cacheName: string) {
-  const result = await topicClient.rotateWebhookSecret(cacheName, 'examples webhook');
-  if (result instanceof RotateWebhookSecret.Success) {
-    console.log('successfully rotated the webhook secret');
-  } else if (result instanceof RotateWebhookSecret.Error) {
-    throw new Error(
-      `An error occurred while attempting to rotate the secret for the webhook 'examples webhook' inside of cache '${cacheName}': ${result.errorCode()}: ${result.toString()}`
-    );
-  }
-}
-
-async function example_API_GetWebhookSecret(topicClient: TopicClient, cacheName: string) {
-  const result = await topicClient.getWebhookSecret(cacheName, 'examples webhook');
-  if (result instanceof GetWebhookSecret.Success) {
-    console.log('successfully retrieved the webhook secret');
-  } else if (result instanceof GetWebhookSecret.Error) {
-    throw new Error(
-      `An error occurred while attempting to fetch the secret for the webhook 'examples webhook' inside of cache '${cacheName}': ${result.errorCode()}: ${result.toString()}`
-    );
-  }
-}
+// async function example_API_ListWebhooks(topicClient: TopicClient, cacheName: string) {
```

Comment:
I had to comment out the example code for webhooks since the CI kept failing for it.
Will publish this sdk version and update the tests accordingly in a follow-up pr.

Reply:
i'm confused as to why these would fail, the package.json in the examples dir should be pinned to a released version of the SDK. Also it's important for the sake of backward compatibility that this syntax still work, so I think we will need to get to the bottom of it before we merge this.